packages/webapp/components/invite/Referral.tsx

```diff
@@ -0,0 +1,227 @@
+          onClick={() => {
+            handleAcceptClick();
+          }}
+          target="_blank"
+          rel={anchorDefaultRel}
+          loading={isLoading}
+          disabled={isLoading || isSuccess}
+        >
+          <FlexCentered className="gap-2 mobileL:typo-title3">
+            <BrowsersIcon
+              width={isMobile ? '58px' : '80px'}
+              height={isMobile ? '30px' : '40px'}
+              className="text-theme-label-primary"
+            />
+            Try it now - It&apos;s free
+          </FlexCentered>
+        </Button>
+      </div>
+      <div className="flex fixed flex-auto w-full h-full laptop:initial laptop:bottom-[unset] bottom-[-49%]">
+        <div className="flex relative laptop:absolute justify-center self-center m-auto mx-2 z-10 laptopL:right-[-6rem] laptop:right-[-7.5rem] w-[fit-content] laptop:w-[initial]">
+          <img
+            src={
+              cloudinary.referralCampaign.genericReferral.purpleEdgeGlowTablet
+            }
+            alt="Purple glow right edge"
+            className="fixed laptop:top-0 w-full laptop:w-auto laptop:h-full bottom-[-1rem] laptop:bottom-[unset] laptop:rotate-[270deg] z-20"
+          />
+          <img
+            src={cloudinary.referralCampaign.genericReferral.appScreenshot}
+            alt="Daily.dev app screenshot of my feed page"
+            className="object-contain w-full laptop:w-auto h-auto laptop:h-full z-10 laptopXL:max-h-[39.5rem] laptop:max-h-[25.35rem]"
+          />
+          <img
+            src={cloudinary.referralCampaign.genericReferral.playButton}
+            alt="Play daily.dev introduction video"
+            className="absolute laptop:self-center laptopL:m-auto laptopXL:h-auto laptop:mr-[8.5rem] laptop:mt-[0.5rem] z-20 laptop:h-[8rem] tablet:h-[14.5rem] h-[7.8rem] tablet:mt-[-1.5rem]"
+          />
+          <button
+            className="absolute z-1 w-full h-full"
+            type="button"
+            aria-label="Play daily.dev introduction video"
+            onClick={() => {
+              setVideoOpen(true);
+            }}
+          />
+        </div>
+      </div>
+      {isVideoOpen && (
+        <Modal
+          // eslint-disable-next-line @dailydotdev/daily-dev-eslint-rules/no-custom-color
+          className="px-8 bg-black"
+          kind={ModalKind.FlexibleCenter}
+          size={ModalSize.XLarge}
+          isOpen={isVideoOpen}
+          onRequestClose={onVideoModalClose}
+        >
+          <CloseButton
+            buttonSize={ButtonSize.Small}
+            className="top-3 right-3 border-white"
+            onClick={onVideoModalClose}
+            position={LogoPosition.Absolute}
+          />
+          <iframe
+            className="w-full border-none aspect-video"
+            src="https://www.youtube.com/embed/igZCEr3HwCg"
+            title="YouTube video player"
+            allow="encrypted-media;web-share"
+            allowFullScreen
+          />
+        </Modal>
+      )}
```

Comment:
I have applied the necessary changes, but the modal still is not showing, probably I've missed something, so it would be great if we can take a look together, thanks! 🙏

Reply:
Sorted, pushed the changes :D